pkg/detector/detector.go

```diff
@@ -1004,6 +1005,7 @@ func (d *ResourceDetector) OnResourceBindingUpdate(_, newObj interface{}) {
 // ReconcileResourceBinding handles ResourceBinding object changes.
 // For each ResourceBinding changes, we will try to calculate the summary status and update to original object
 // that the ResourceBinding refer to.
+//nolint:gocyclo
```

Comment:
Do we still need to ignore this checking? As the complexity is decreased.

Reply:
Oh I forgot to delete this, we don't need it now.